README.md

```diff
@@ -75,6 +76,8 @@ var myOverlay = new Overlay('myOverlay', {
 * `zindex`: String. Value of the CSS z-index property of the overlay. _Default set via CSS_: '10'
 * `preventclosing`: Boolean. Prevents closure of overlay via standard x button or escape key. For use when you are directing the user to somewhere else. Only valid with modal set to true.
 * `customclose`: Boolean. If you do not use the header, but want to provide a close button in your html / src (with a class of `o-overlay__close`), setting customclose to true will attach o-overlay's close handler function to that button.
+* `parentNode`: String. Should be a query selector for a DOM element. If set, the overlay will be appended as a child of this rather than the document body or target.
+* `nested`: Boolean. If set to true, the resize listeners will not be set up. This boolean should be used in conjunction with the `parentNode` setting to allow an overlay to be positioned within a DOM element rather than overlaid on top of everything. Defaults to false.
```

Comment:
Yes.
parentNode without nested would just attach the overlay div to whatever the first selector did
Nested without parentNode would append the overlay to the body, but without any of the positioning / resize logic so it would appear at the bottom of the body
Neither of these are particularly useful without the other but they are not absolutely linked. One is about position within the document and one is about style